packages/@aws-cdk/aws-scheduler-alpha/lib/schedule.ts

```diff
@@ -1,6 +1,100 @@
-import { IResource } from 'aws-cdk-lib';
+import { IResource, Resource } from 'aws-cdk-lib';
+import { CfnSchedule } from 'aws-cdk-lib/aws-scheduler';
+import { Construct } from 'constructs';
+import { IGroup } from './group';
+import { ScheduleExpression } from './schedule-expression';
+import { IScheduleTarget } from './target';
 
 /**
  * Interface representing a created or an imported `Schedule`.
  */
-export interface ISchedule extends IResource {}
+export interface ISchedule extends IResource {
+  readonly scheduleName: string;
+  readonly group?: IGroup;
+  readonly scheduleArn: string;
+}
+
+/**
+ * Construction properties for `Schedule`.
+ */
+export interface ScheduleProps {
+  /**
+   * The expression that defines when the schedule runs. Can be either a `at`, `rate`
+   * or `cron` expression.
+   */
+  readonly schedule: ScheduleExpression;
+
+  /**
+   * The schedule's target details.
+   */
+  readonly target: IScheduleTarget;
+
+  /**
+   * The name of the schedule.
+   *
+   * Up to 64 letters (uppercase and lowercase), numbers, hyphens, underscores and dots are allowed.
+   *
+   * @default - A unique name will be generated
+   */
+  readonly scheduleName?: string;
+
+  /**
+   * The description you specify for the schedule.
+   *
+   * @default - no value
+   */
+  readonly description?: string;
+
+  /**
+   * The schedule's group.
+   *
+   * @deafult - By default a schedule will be associated with the `default` group.
+   */
+  readonly group?: IGroup;
+}
+
+/**
+ * An EventBridge Schedule
+ */
+export class Schedule extends Resource implements ISchedule {
```

Comment:
@kaizencc I have moved `Schedule` class out of private zone. Do we need to add a note to readme or anything?

Reply:
Added now